public/member_ops.js

```diff
@@ -153,7 +153,35 @@ function handle_members(event) {
         }
       });
     } else if (event.target === deleteMember) {
-    // Your code goes here!
+      let headers = { "Content-Type": "application/json" };
+      let csrf_cookie = getCookie("CSRF-TOKEN");
+      if (csrf_cookie) {
+        headers["X-CSRF-Token"] = csrf_cookie;
+      }
+      fetch(`${members_path}/${memberId3.value}`, {
+        method: "DELETE",
+        headers: headers,
+      })
+        .then((response) => {
+          if (response.status === 200) {
+            response.json().then((data) => {
+              resultsDiv.innerHTML = "";
+              let parag = document.createElement("P");
+              parag.textContent = JSON.stringify(data);
+              resultsDiv.appendChild(parag);
+            });
+          } else {
+            response.json().then((data) => {
+              alert(
+                  `Return code ${response.status} ${response.statusText} ${JSON.stringify(data)}`
+              );
+            });
+          }
+        })
+        .catch((error) => {
+          console.log(error);
```

Comment:
I'm curious why you chose to include this console.log()?  If you have this in here for debugging purposes, another option is `console.error();` rather than `console.log(error);`.  `console.error();` will log a stack trace to the console as well.

Reply:
Yes, console.error(); is more appropriate. Thank you